packages/cli/src/utils.ts

```diff
@@ -24,3 +24,6 @@ export function tail(id: string, name: string): ChildProcess {
     }
   );
 }
+export function isUrl(url: string) {
+  return /^https?:\/\/.+/.test(url)
```

Comment:
How about Node.js `url.parse()`?

Reply:
This PR supports http and https only, `url.parse()` is out of the protocols.

Reply:
`url` can handle http/https, try `urlobj.protocol`.